man/journalctl.xml

```diff
@@ -87,18 +87,15 @@
       causes all matches before and after to be combined in a
       disjunction (i.e. logical OR).</para>
 
-      <para>As shortcuts for a few types of field/value matches, file
-      paths may be specified. If a file path refers to an executable
-      file, this is equivalent to an <literal>_EXE=</literal> match
-      for the canonicalized binary path. Similarly, if a path refers
-      to a device node then match is added for the kernel name of the
-      device (<literal>_KERNEL_DEVICE=</literal>). Also, matches for the
-      kernel names of all the parent devices are added automatically.
-      Device node paths are not stable across reboots, therefore match
-      for the current boot id (<literal>_BOOT_ID=</literal>) is
-      always added as well. Note that only the log entries for
-      the existing device nodes maybe queried by providing path to
-      the device node.</para>
```

Comment:
> (I haven't force pushed the revision as I believe that will overwrite the comments.)

github changed behaviour a few months back, comments on old revisions are not lost anymore. It's fine to push.

> The path may be a regular file or a link to a file and the file must exist at the time of the query.

This isn't as clear as the previous sentence ("Note that only the log entries for the existing device nodes maybe queried by providing path to the device node."), because a device node is neither a regular file nor a link. We are not trying to make the text as concise as possible — brevity is not a virtue on its own. Can you just leave this final sentence as it was? Even if strictly speaking it is redundant, it makes it easier to understand a corner case.